app/models/concerns/hud_concerns/enrollment_shared.rb

```diff
@@ -0,0 +1,29 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HudConcerns::EnrollmentShared
```

Comment:
This naming isn't consistent with other modules we have in `app/models/concerns`. I think more consistent (and less redundant) would be simply `Hud::Enrollment`?

We're not super consistent across the board, we do have one module called `UserConcern`, but that seems redundant to me.

Reply:
I think we want to avoid the same namespace, Rails often gets confused, so let's go with `HudConcerns::Enrollment` for now.

Reply:
Done